jipt/src/ui/styles.ts

```diff
@@ -0,0 +1,47 @@
+/* Inline styles are used everywhere so that it does not conflict with parent window styles */
```

Comment:
Might be worth it to use shadow DOM for this at some point, but i think this solution is quite simple and good enough 👌